src/exchanges/crdtExchange.ts

```diff
@@ -464,6 +454,64 @@ class CrdtManager {
     }
   }
 
+  private persistMutations() {
+    console.log('persistMutations', this.#mutations);
+    const mutationOps = Array.from(this.#mutations.values());
+
+    const persistedMutations = mutationOps.map((mut) => {
+      return {
+        query: print(mut.query),
+        variables: mut.variables,
```

Comment:
Do we need to persist context (or some subset of it)?

Reply:
So far, it hasn't appeared to be necessary to persist the context for this client to function. However, if there's anything we want to persist, we could.

Reply:
I'm thinking that we need to persist `context[PROCESSED_OPERATION_KEY]` so that timestamps exchange doesn't re-timestamp if we ever switch to sending the mutation back through the client.

There may be other things specific to apm-mobile if other exchanges depend on the original context that a mutation had.

It's probably best to not persist the _whole_ context blindly though, as it also contains a URL, among other things we may want to update. Imagine if we change the URL in a new app release but old persisted mutations kept the old URL - they could fail.

Reply:
If we want to use the standard urql interface for StorageAdapter, then it expects metadata to be a `SerializedRequest`, which does not have context as part of its interface. If we move away from using the StorageAdapter and SerializedRequest interfaces from urql, then we can do this. However, it was expressed to me by you and Malcolm that we would like to adhere to these types from urql.